bootstrap/service.go

```diff
@@ -262,26 +307,90 @@ func (bs bootstrapService) UpdateConnections(ctx context.Context, token, id stri
 			return ErrThings
 		}
 	}
-	if err := bs.configs.UpdateConnections(ctx, owner, id, channels, connections); err != nil {
+	if err := bs.configs.UpdateConnections(ctx, user.GetDomainId(), id, channels, connections); err != nil {
 		return errors.Wrap(errUpdateConnections, err)
 	}
 	return nil
 }
 
+func (bs bootstrapService) listClientIDs(ctx context.Context, userID string) ([]string, error) {
+	tids, err := bs.auth.ListAllObjects(ctx, &magistrala.ListObjectsReq{
+		SubjectType: auth.UserType,
+		Subject:     userID,
+		Permission:  auth.ViewPermission,
+		ObjectType:  auth.ThingType,
+	})
+	if err != nil {
+		return nil, errors.Wrap(svcerr.ErrNotFound, err)
+	}
+	return tids.Policies, nil
+}
+
+func (bs bootstrapService) filterAllowedThingIDs(thingIDs []string) ([]string, error) {
+	var ids []string
+	for _, thingID := range thingIDs {
+		for _, tid := range thingIDs {
+			if thingID == tid {
+				ids = append(ids, thingID)
+			}
+		}
+	}
+	return ids, nil
+}
```

Comment:
This function work doesn't make sense. 
Please verify 
Do we need filter Allowd ThingsIDs ?

Reply:
@arvindh123 on second check, since `bs.auth.ListAllObjects` is already called, we don't need to `filterAllowedThingsIDs`